test/zkbob/ZkBobPool.t.sol

```diff
@@ -231,11 +377,11 @@ abstract contract AbstractZkBobPoolTest is AbstractMainnetForkTest {
     function testAppendDirectDeposits() public {
         _setUpDD();
 
-        vm.prank(user1);
-        _transferAndCall(10 ether, user2, zkAddress);
-
-        vm.prank(user1);
-        _transferAndCall(5 ether, user2, zkAddress);
+        vm.startPrank(user1);
+        IERC20(token).approve(address(queue), 100 ether / D);
```

Comment:
Before it was tested that `transferAndCall` works  for BOB token without prior `approve`. Does it make sense to preserve the test?